packages/block-editor/src/components/url-popover/test/index.js

```diff
@@ -22,8 +32,11 @@ describe( 'URLPopover', () => {
 			</URLPopover>
 		);
 
-		// wait for `Popover` effects to finish
-		await act( () => Promise.resolve() );
+		await waitFor( () =>
+			expect(
+				getWrappingPopoverElement( screen.getByText( 'Editor' ) )
+			).toBePositionedPopover()
+		);
```

Comment:
One interesting thing here is that the test doesn't pass an `anchor` prop to the `URLPopover`, so there should be no positioning.

However, the `Popover` component has a fallback for that case, where it will use the popover element's parent as anchor. @ciampo Are there cases where the fallback is actually used and does something useful? I can imagine using a popover inside a paragraph for example:
```js
<p>Lorem ipsum <span>dolor<Popover>latin for pain</Popover></span> sit amet<p>
```
and the it will be permanently attached to the span with word "dolor"? Is that what the fallback element is for?

If we didn't do any fallback for `anchor`, then it would remain `null` and `floating-ui` wouldn't do any positioning at all. The popover would have a `position: absolute` style and that's all.